logic/src/commonMain/kotlin/com/wire/kalium/logic/data/featureConfig/FeatureConfigMapper.kt

```diff
@@ -152,3 +153,17 @@ class FeatureConfigMapperImpl : FeatureConfigMapper {
             fromModel(model.status)
         )
 }
+
+fun MLSMigrationModel.toEntity(): MLSMigrationEntity =
+    MLSMigrationEntity(
+        status = status.equals(Status.ENABLED),
+        startTime = startTime,
+        endTime = endTime
+    )
+
+fun MLSMigrationEntity.toModel(): MLSMigrationModel =
+    MLSMigrationModel(
+        status = if (status) Status.ENABLED else Status.DISABLED,
```

Comment:
question: isn't the status itself boolean?

Reply:
No, it's an enum for some reason.